README.md

```diff
@@ -23,6 +23,13 @@ Install Java 8. We recommend installing IntelliJ to edit files, but any editor s
 1. Run the following command in terminal: ```java -jar target/LiUni_Onboarding-1.0-SNAPSHOT.jar server```
 2. In a separate terminal, run ```curl --data "MESSAGE" http://localhost:8080/api/1.0/twitter/tweet``` where "MESSAGE" is what it is you would like to post.
 
+## Testing the Lithium University Onboarding Program ##
+First, navigate to the directory the project is in within terminal.
+### Running an individual set of tests ###
+Run: ```mvn test -Dtest=<Java Test Class Name>```
+### Running all tests ###
+Run: ```mvn test -Dtest="*Tests"```
```

Comment:
you shouldn't need the command line argument to run all the tests

Reply:
Im not sure I understand what you mean by this comment?